jython/LogixNG/ExportLogixNG.py

```diff
@@ -124,7 +124,7 @@
     # Write each line from the "lines" list to the output file, finish with the final xml line.
     with open(outputFileName, 'w') as outfile:
         for line in lines:
-            outfile.write(line.replace(':AUTO:', '$' + newKeyword + ':'))
+            outfile.write(line.decode("utf-8").replace(':AUTO:', '$' + newKeyword + ':').encode("utf-8"))
         outfile.write('</layout-config>\n')
 
     log.info('Export completed, AUTO replaced by {}, file {} has been created'.format(newKeyword, outputFileName))
```

Comment:
This line also needs to be fixed.
```
javax.script.ScriptException: UnicodeEncodeError: 'ascii' codec can't encode character u'\xe8' in position 1: ordinal not in range(128) in <script> at line number 130
```

Reply:
is it possible for the xml panel file not to be encoded in utf-8?